src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/AgencyInternalLocation.kt

```diff
@@ -11,6 +11,9 @@ import jakarta.persistence.ManyToOne
 import jakarta.persistence.SequenceGenerator
 import jakarta.persistence.Table
 import org.hibernate.Hibernate
+import org.hibernate.annotations.JoinColumnOrFormula
+import org.hibernate.annotations.JoinColumnsOrFormulas
+import org.hibernate.annotations.JoinFormula
```

Comment:
it would be goof to add `@EntityOpen`  to this class so we can generate a proxy, I think this class ends up never being lazy loaded possibly?

Reply:
👍 already has it .